packages/terra-layout/src/terra-dev-site/test/layout/TestLayoutCommon.scss

```diff
@@ -0,0 +1,28 @@
+// stylelint-disable scss/at-mixin-pattern
+
+:local {
+  @mixin content-common($background) {
```

Comment:
lets remove this mixin and add the styles directly

Reply:
Done